examples/vector_databases/azuresearch/Getting_started_with_azure_cognitive_search_and_openai.ipynb

```diff
@@ -0,0 +1,754 @@
518218517303, -0.017751257866621017, 0.018605664372444153, -0.005177450366318226, 0.000956425559706986, -0.017406944185495377, 0.05707947537302971, 0.02359182946383953, 0.0059107099659740925, -0.004303914960473776, 0.005104124546051025, -0.033666178584098816, -0.0009165744995698333, 0.006771492771804333, 0.009551502764225006, 0.011406968347728252, -0.007951084524393082, -0.012229494750499725, 0.010042468085885048, 0.02876928076148033, 0.033793702721595764, 0.004877770319581032, 0.011432473547756672, 0.018325112760066986, -0.015302808955311775, -0.006969154346734285, -0.0037651285529136658, -0.0160807017236948, -0.016590794548392296, 0.0318298414349556, 0.04139409959316254, -0.02448449470102787, -0.017279421910643578, 0.0009372970671392977, -0.005282657220959663, 0.011075408197939396, 0.0003893449029419571, -0.006618464831262827, -0.014575925655663013, 0.0030573736876249313, 0.014116841368377209, 0.013645004481077194, -0.021895768120884895, -0.004613159224390984, 0.01791703887283802, 0.0321614034473896, 0.009723659604787827, -0.015723636373877525, -0.018656674772500992, -0.019562089815735817, -0.00468967342749238, -0.006108371075242758, -0.0045525855384767056, -0.01699887029826641, -0.008352783508598804, -0.0033379250671714544, 0.019370805472135544, 0.023630088195204735, 0.026983952149748802, 0.015060514211654663, -0.04397007077932358, -0.00043557214667089283, 0.0026748033706098795, -0.004093501251190901, -0.007549385540187359, 0.01308390125632286, -0.016118958592414856, 0.0013485599774867296, 0.043434470891952515, -0.027519552037119865, 0.012599312700331211, -0.008269893005490303, 0.047821275889873505, 0.012363394722342491, 0.016169967129826546, 0.0036152885295450687, 0.014524916186928749, -0.006385734770447016, -0.017113640904426575, -0.0008655651472508907, -0.011936190538108349, -0.018146580085158348, -0.004405933897942305, 0.019664108753204346, 0.022074300795793533, 0.049479082226753235, 0.02879478596150875, -0.035323984920978546, 0.02887129969894886, -0.006372982170432806, 0.029075337573885918, 0.012082843109965324, -0.00911792367696762, -0.0062103900127112865, -0.018108323216438293, -0.017661992460489273, -0.005014857742935419, 0.00018431116768624634, -0.01938355714082718, -0.03833353519439697, 0.005824631545692682, -0.02093934267759323, -0.0020180579740554094, -0.018401628360152245, 0.005990412086248398, 0.011974448338150978, -0.01449941098690033, 0.0213856752961874, 0.011636510491371155, -0.01831236109137535, -0.019944660365581512, 0.012930873781442642, -0.013951060362160206, -0.008569573052227497, -0.019791632890701294, 0.003940473310649395, -0.010131734423339367, -0.023681096732616425, -0.008193379268050194, 0.006516445893794298, -0.00263017020188272, -0.00646862480789423, -0.01623372919857502, 0.015302808955311775, 0.0245865136384964, -0.023757610470056534, -0.02056952565908432, -0.01583840698003769, -0.009277327917516232, -0.008442049846053123, 0.0007037697941996157, 0.02059503085911274, -0.019855394959449768, -0.009181685745716095]\n"
```

Comment:
Printing the whole vector doesn't seem necessary, perhaps the first few items would be good enough.

Reply:
cleaned up!